app/Http/Controllers/Api/v1/GameController.php

```diff
@@ -9,7 +9,7 @@
 
 class GameController extends Controller
 {
-    public function index(){
+    public function index() {
```

Comment:
__[code_sniffer]__
* Opening brace should be on a new line

<!-- Comment by [SideCI](https://sideci.com) -->